src/screens/Wallet/Send/SendContainer.js

```diff
@@ -8,26 +8,385 @@ import {
+        recipientAddress: recipient,
+        selectedCurrency,
+        currentToken: selectedToken.symbol,
+        amountToSend: {
+          token: amount,
+          currency: amountInCurrency,
+        },
+        fee: {
+          token: gasFeeAmount,
+          currency: gasFeeAmountInCurrency,
+        },
+        balance: {
+          token: amount,
+          currency: firtsAvailableToken.balance,
+        },
+        feeMultiplier,
+      }
+
+      const tx = {
+        to: passProps.recipientAddress,
+        from: address,
+        value: convertBTCToSatoshi(passProps.amountToSend.token),
+      }
+
+      requestBitcoinUtxoByAddress(address)
+        .then((results) => {
+          console.log("RESULTS: ", results)
+          console.log("networks: ", networks)
+          if (results && results.payload.data) {
+            const feeRate = passProps.fee.token
+            // const prepared = await dispatch(BitcoinUtils.prepareBitcoinTransaction(tx, token, network, utxos))
+            prepareBitcoinTransaction({
+              tx,
+              blockchain,
+              feeRate,
+              network: networks.testnet,
+              utxos: results.payload.data,
+            })
+              .then((transaction) => {
+                console.log('HERE IS TRANSACTION: ', transaction)
+                console.log('HERE IS TRANSACTION.buildIncomplete().toHex(): ', transaction.prepared.buildIncomplete().toHex())
+                const unsignedTxHex = transaction.prepared.buildIncomplete().toHex()
+                Keychain.getInternetCredentials(parentAddress)
+                  .then((keychain) => {
+                    console.log("PASsWORD: ", keychain.password)
+                    decryptWallet(currentWallet.encrypted, keychain.password)
+                      .then((decrypted) => {
+                        console.log("decrypted: ", decrypted)
+                        const signedTX = signTransaction({ unsignedTxHex, network: networks.testnet, privateKey: decrypted.privateKey })
+                        console.log("SIGNED: ", signedTX)
+                      })
+                  })
+              })
+          }
+        })
+
+      this.setState({ passProps }, () => this.handleTogglePasswordModal())
+
+    } else {
+      Alert.alert('Input error', 'Please fill address and amount', [
+        { text: 'Ok', onPress: () => { }, style: 'cancel' },
+      ])
+    }
+  }
+
+  handleChangeRecipient = (name, value) => {
+    if (typeof value === 'string') {
+      const { blockchain } = this.props.navigation.state.params
+      const {
+        isAmountInputValid,
+        recipient,
+        amount,
+      } = this.state
+      // Check for Ethereum
+      let dummyValidationOfRecipientInput =
+        value !== null &&
+        value !== '' &&
+        (value.length >= 40 || value.length <= 44) &&
+        value.startsWith('0x')
+
+      // Check for BitCoin-based
+      if (blockchain !== DUCK_ETHEREUM) {
```

Comment:
DUCK_ETHEREUM must not be used like this. Here we need blockchain's name BLOCKCHAIN_ETHEREUM from  packages/ethereum/constants.js